CHANGELOG.md

```diff
@@ -9,7 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+* Added `BRep.trim` as in-place variation of `slice`.
```

Comment:
trim and slice are not the same.
in any case:
* trim (in-place) + trimmed (return trimmed copy)
* slice (in-place) + sliced (return sliced copy)

Reply:
slicing is typically an operation with a plane as an operand.

trimming is typically with a boundary curve/surface as an operand. it is basically like adding a different boundary to the otherwise unbounded underlying geometries...

Reply:
updated the comment as there is now distinction between `split`, `slice` and `trim`.